code/frontend/.mocharc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
https://typestrong.org/ts-node/docs/recipes/mocha/ is the official recipe for typescript + mocha. 

We need ts-node for mocha to work with typescript. We are setting the loader to ESM so that it works with the ECMAScript modules in our codebase.